### ISSUE TITLE
Accessibility Barrier: 2.4.5 Multiple Ways

### ISSUE THREAD

**Issue by kimviens** [gc-da11yn/gc-da11yn.github.io#592]
### Describe the purpose of this issue

I have been struggling to find the information I need on the Digital Accessibility Toolkit website for months. I often avoid using it because it is too frustrating. For example, today I wanted to share the link to the page about the Access Working Group, which I know we had before. I checked [Digital accessibility in the Government of Canada](https://a11y.canada.ca/en/digital-accessibility-in-the-government-of-canada) > [Community Directory](https://a11y.canada.ca/en/community-directory/), but it was not there. The website layout and the link labels are so unclear that I cannot tell if the page still exists or not.

The problem here is that I have no other way to find information:
- The site menu is for the Canada.ca site not digital accessibility toolkit
- The search bar is for Canada.ca so when I try to search the DAT I get 121,346 results.
- The breadcrumbs are not useful because content is spread out and the categories are not well described. For example, I never would have guessed that [Create web content](https://a11y.canada.ca/en/create-web-content) would lead to instructions on how to create multimedia content.

A site menu or a search bar for the DAT would make the website more usable for me. Right now, it is very hard to find the right content for my clients. I have saved some important pages, but I don't want to waste time browsing the website. I also don't know if the content I need is missing or hidden in some obscure location.

I appreciate your attention to my concern.

**Comment by kimviens** [comment]
@JulieBB23 If you have a moment, I would appreciate your thoughts on this.

**Comment by JulieBB23** [comment]
Thanks for sharing, we will review and assess the current information architecture and conduct some user testing and survey. As for the AAACT, i do think it was removed because the content was just outdated and we are waiting from them to send a new and updated content @shawnthompson  can you confirm on that ?

As for your question and concerns regarding the following below, I think it is constrain that we have to follow but again @shawnthompson  could you confirm please?
- The site menu is for the Canada.ca site not digital accessibility toolkit
- The search bar is for Canada.ca so when I try to search the DAT I get 121,346 results.
- The breadcrumbs are not useful because content is spread out and the categories are not well described. For example, I never would have guessed that [Create web content](https://a11y.canada.ca/en/create-web-content) would lead to instructions on how to create multimedia content.

**Comment by shawnthompson** [comment]
@kimviens thanks for bringing up your concerns. We are aware the structure of the site isn't ideal. We are working on tagging the pages and allowing for users to find the same page in different avenues, like by subject, by their role, by topics. @JulieBB23 has already put an issue in to start tagging the pages by role: [[CHANGE] Tagging all the content by role · Issue #582 · gc-da11yn/gc-da11yn.github.io](https://github.com/gc-da11yn/gc-da11yn.github.io/issues/582). Just need the help to work on that. Feel free, if you have the 🥄 spoons to have a look at the Excel file in that issue and see if you agree with the roles to those pages. 

As for the other concerns listed:

- The site menu is for the Canada.ca site not digital accessibility toolkit
  - This is something we can not change, it's in the rules for using the Canada.ca theme 
- The search bar is for Canada.ca so when I try to search the DAT I get 121,346 results.
  - This is waiting for SSC to approve the results page to use the SSC name because we can not have a results page directly under Canada.ca. It has to go to Canada.ca > Shared Services Canada > Digital Accessibility Toolkit. Once that's done, we will have a results page on Canada.ca which will list a11y.canada.ca domain results first.
- The breadcrumbs are not useful because content is spread out and the categories are not well described. For example, I never would have guessed that [Create web content](https://a11y.canada.ca/en/create-web-content) would lead to instructions on how to create multimedia content.
  - The breadcrumbs are never useful because they are not how you get somewhere (according to the same rules as the search), they are only by subject. So every pages will be assigned one subject but can have multiple topics.
  - Example: [Captioning Checklist - Digital Accessibility Toolkit](https://a11y.canada.ca/en/captioning-checklist/), will have a subject of "[Create web content](https://a11y.canada.ca/en/create-web-content)" but can have topics for _Audio_ and _Video_, which would lead to pages listing everything "tagged" with those topics.

I hope these answer some of your concerns. Please stay tuned as we work on improving our site.

I'd like to move this issue to discussions, @kimviens please let me know if you have any objections to doing this.

Thanks for your support
